datasets/freebase_qa/README.md

```diff
@@ -0,0 +1,165 @@
+## Dataset Creation
+ 
+### Curation Rationale
+ 
+[More Information Needed]
+ 
+### Source Data
+ 
+#### Initial Data Collection and Normalization
+ 
+The data set is generated by matching trivia-type question-answer pairs with subject-predicateobject triples in Freebase. For each collected question-answer pair, we first tag all entities in each question and search for relevant predicates that bridge a tagged entity with the answer in Freebase. Finally, human annotation is used to remove false positives in these matched triples. 
+ 
+#### Who are the source language producers?
+ 
+[More Information Needed]
+ 
+### Annotations
+ 
+#### Annotation process
+ 
+[More Information Needed]
+ 
+#### Who are the annotators?
+ 
+[More Information Needed]
+ 
+### Personal and Sensitive Information
+ 
+[More Information Needed]
+ 
+## Considerations for Using the Data
+ 
+### Social Impact of Dataset
+ 
+[More Information Needed]
+ 
+### Discussion of Biases
+ 
+[More Information Needed]
+ 
+### Other Known Limitations
+ 
+[More Information Needed]
+ 
+## Additional Information
+ 
+### Dataset Curators
+ 
+Kelvin Jiang - Currently at University of Waterloo. Work was done at
+York University.
+ 
+### Licensing Information
+ 
+[More Information Needed]
+ 
+### Citation Information
+ 
+```
+@inproceedings{jiang-etal-2019-freebaseqa,
+    title = "{F}reebase{QA}: A New Factoid {QA} Data Set Matching Trivia-Style Question-Answer Pairs with {F}reebase",
+    author = "Jiang, Kelvin  and
+      Wu, Dekun  and
+      Jiang, Hui",
+    booktitle = "Proceedings of the 2019 Conference of the North {A}merican Chapter of the Association for Computational Linguistics: Human Language Technologies, Volume 1 (Long and Short Papers)",
+    month = jun,
+    year = "2019",
+    address = "Minneapolis, Minnesota",
+    publisher = "Association for Computational Linguistics",
+    url = "https://www.aclweb.org/anthology/N19-1028",
+    doi = "10.18653/v1/N19-1028",
+    pages = "318--323",
+    abstract = "In this paper, we present a new data set, named FreebaseQA, for open-domain factoid question answering (QA) tasks over structured knowledge bases, like Freebase. The data set is generated by matching trivia-type question-answer pairs with subject-predicate-object triples in Freebase. For each collected question-answer pair, we first tag all entities in each question and search for relevant predicates that bridge a tagged entity with the answer in Freebase. Finally, human annotation is used to remove any false positive in these matched triples. Using this method, we are able to efficiently generate over 54K matches from about 28K unique questions with minimal cost. Our analysis shows that this data set is suitable for model training in factoid QA tasks beyond simpler questions since FreebaseQA provides more linguistically sophisticated questions than other existing data sets.",
+}
+```
```

Comment:
````suggestion
```
### Contributions

Thanks to [@gchhablani](https://github.com/gchhablani) for adding this dataset.
````
We have a new section now ;)